docs/components/input-stepper/use-cases.md

```diff
@@ -49,6 +49,41 @@ Use `min` and `max` attribute to specify a range.
 ></lion-input-stepper>
 ```
 
+### Value text
+
+Use the `.valueText` property to override the value with a text.
+
+```js preview-story
+export const valueText = () => {
+  const values = [
```

Comment:
Since an input step will always have a set of discrete steps and labels (value-texts) belonging to it, maybe a better api would be an index based array corresponding with the amount of steps 

Otherwise you would have to do this always (there's no use case for providing just one valueText